test/PostDecoder.elm

```diff
@@ -5,6 +5,7 @@ decodePost : Json.Decode.Decoder Post
 decodePost =
   Json.Decode.succeed Post
     |: ("id" := Json.Decode.int)
+    |: ("pointless" := Json.Decode.succeed ())
```

Comment:
Hmm...this can't be right. The decode should fail on an object that's:

```json
{
  ...
  pointless: "Unexpected string."
  ...
}
```

...but this one will silently succeed.